### ISSUE TITLE
Can't drag if there's a touchable

### ISSUE THREAD

**Issue by mqtik** [jacklein/rn-bottom-drawer#28]
The component has conflicts with any touchables if you want to drag up/down and still have a button/textinput on the header.

**Comment by shubymao** [comment]
Yes, I would also like to see this problem being fixed. My scrollable component inside the drawer should be interactable instead of triggering the drawer scrolling. Please provide feedback.

**Comment by GeorgiySurkov** [comment]
Yeah, I have a `TextInput` in the drawer header and I can't drag the drawer because of it. I've tried using `TouchableOpacity` but it doesn't help.

**Comment by MinhazMM** [comment]
Any fix on this?
